app/services/participants/sync_dqt_induction_start_date.rb

```diff
@@ -2,7 +2,7 @@
 
 module Participants
   class SyncDQTInductionStartDate < BaseService
-    FIRST_2021_ACADEMIC_DATE = Date.new(2021, 9, 1)
+    FIRST_2021_ACADEMIC_DATE = Cohort::INITIAL_COHORT_START_DATE
```

Comment:
I was going to comment and ask if `INITIAL_COHORT_START_DATE` was a bit wordy - but I can't think of anything better